packages/shared/lib/services/connection.service.ts

```diff
@@ -985,6 +985,8 @@ class ConnectionService {
         });
         await logCtx.info('App connection was approved and credentials were saved');
 
+        await logCtx.success();
```

Comment:
Can you remove this one too, success is applied at a higher level

Reply:
Done with https://github.com/NangoHQ/nango/pull/2300/commits/6ecbe5c2badf1952121fa4c5a2a28620de0dc281